test/chi/components/header.pug

```diff
@@ -144,16 +194,16 @@ h3 Header with Tabs and Buttons
     nav.chi-header__content
       .chi-header__brand.-flex--fill.-flex-lg--grow0
         a.chi-brand(href="#", aria-label="CenturyLink")
-          svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg', aria-hidden="true")
```

Comment:
@sabah1910 please remove Header w/ tabs and buttons example as it no longer exists in the doc.